### ISSUE TITLE
Add routerLink to ion item?

### ISSUE THREAD

**Issue by Tommertom** [ionic-team/ionic-docs#1852]
**URL**
The URL at which the content is missing or inaccurate
https://ionicframework.com/docs/api/item

**What is missing or inaccurate about the content on this page?**
I believe it would be better to add explicitly api doc on usage of routerLink for ion-item, besides only mentioning routerDirection and routerAnimation

**Comment by tobijafischer** [comment]
I agree. As somebody quite new to Ionic, it would have helped me, if there was some information about the routerLink attribute (in the ion-card component).
